packages/yourloops/components/dialogs/medical-record-edit-dialog.tsx

```diff
@@ -198,17 +164,19 @@ export default function MedicalRecordEditDialog(props: MedicalRecordEditDialogPr
         >
           {t('cancel')}
         </Button>
-        <ProgressIconButtonWrapper inProgress={inProgress}>
-          <Button
-            variant="contained"
-            color="primary"
-            disableElevation
-            disabled={disabled}
-            onClick={saveMedicalRecord}
-          >
-            {t('save')}
-          </Button>
-        </ProgressIconButtonWrapper>
+        {user.role !== UserRoles.patient &&
```

Comment:
```suggestion
        {!user.isUserPatient() &&
```

Reply:
Be careful, this will break your tests.


```
    (authHookMock.useAuth as jest.Mock).mockImplementation(() => {
      return {
        user: { role: UserRoles.hcp, isUserPatient: () => false } as User
      }
    })
```

```

    (authHookMock.useAuth as jest.Mock).mockImplementation(() => {
      return { user: { role: UserRoles.patient, isUserPatient: () => true } as User }
    })
```